src/NuGet.Core/NuGet.Commands/PackagesLockFileBuilder.cs

```diff
@@ -57,11 +57,19 @@ public PackagesLockFile CreateNuGetLockFile(LockFile assetsFile)
                     var framework_dep = framework?.Dependencies.FirstOrDefault(
                         dep => StringComparer.OrdinalIgnoreCase.Equals(dep.Name, library.Name));
 
+                    CentralPackageVersion centralPackageVersion = null;
+                    var libraryVersionIsCentrallyDefined = framework?.CentralPackageVersions.TryGetValue(library.Name, out centralPackageVersion);
```

Comment:
Could you use an actual type instead of `var`?

Reply:
Updated.